### PR TITLE
Add OSC_POSITION option in spacemouse input_utils

### PR DESCRIPTION
**Pull request by zhuyifengzju**
#205

### CODE REVIEW

File: robosuite/utils/input_utils.py [hunk 1]
```diff
@@ -245,6 +245,8 @@ def input2action(device, robot, active_arm="right", env_configuration=None):
         # Scale rotation for teleoperation (tuned for OSC) -- gains tuned for each device
         drotation = drotation * 50 if isinstance(device, SpaceMouse) else drotation * 1.5
         dpos = dpos * 125 if isinstance(device, SpaceMouse) else dpos * 75
+    elif controller.name== "OSC_POSITION":
```
Comment (yukezhu) [inline robosuite/utils/input_utils.py:248]: Please follow the coding convention and add an extra space before ==.

Comment (zhuyifengzju) [reply robosuite/utils/input_utils.py]: Updated.

**Comment by yukezhu** [comment]
@zhuyifengzju This PR has merge conflicts with the latest master branch. Could you resolve them so we can approve this?

**Comment by zhuyifengzju** [comment]
Just rebased to the master branch. I think it's up-to-date now.

**Review by yukezhu — APPROVED** [review]
LGTM

**Pull request merged**